### PR TITLE
Add date picker test app

### PR DESCRIPTION
**Pull request by schloerke**
Related: https://github.com/schloerke/shinyjster/pull/38

### CODE REVIEW

File: apps/209-datepicker/app.R [hunk 1]
```diff
@@ -0,0 +1,158 @@
+### Keep this line to manually test this shiny application. Do not edit this line; shinycoreci::::is_manual_app
+
+library(shiny)
+library(magrittr)
+library(rlang)
+
+options(shiny.minified = FALSE)
```
Comment (cpsievert) [inline apps/209-datepicker/app.R:7]: Do you want to reset this option `onStop()`?

**Review by cpsievert — APPROVED** [review]
LGTM assuming GHA tests look good

(sorry I messed things up trying to get bslib/thematic installed for https://github.com/rstudio/shinycoreci-apps/pull/73)

File: apps/209-datepicker/app.R (hunk 1 above)
Comment (wch) [reply apps/209-datepicker/app.R:7]: Oh, this line actually isn't needed.

**Review by wch — APPROVED** [review]

File: apps/209-datepicker/app.R (hunk 1 above)
Comment (schloerke) [inline apps/209-datepicker/app.R:7]: Remove unnecessary line

```suggestion
```

**Pull request merged**